### ISSUE TITLE
hds2202s 

### ISSUE THREAD

**Issue by mikhailmihalkov** [SimoneAlbano000/OWON-HDS2202S-Bode-Plots#1]
Good afternoon ! Tell me the button to start measurements is not active; the program itself sees the device. Is this not a modification of the program?

**Comment by SimoneAlbano000** [comment]
> Good afternoon ! Tell me the button to start measurements is not active; the program itself sees the device. Is this not a modification of the program?

Hi! The button to start the measurements will become active after all the settings are configured correctly.

**Comment by mikhailmihalkov** [comment]
I tried everything, the start measurement button is not active

**Comment by zentribe** [comment]
I had this same issue and turned out it was due to driver issues, solved it by either running previous release or reinstalling drivers for owon. Yours

**Comment by SimoneAlbano000** [comment]
> I had this same issue and turned out it was due to driver issues, solved it by either running previous release or reinstalling drivers for owon. Yours
> 
> Sent from Proton Mail for iOS
> 
> 
> On Sun, Mar 10, 2024 at 7:15 a.m., Simone Albano ***@***.***> wrote:
> 
> > > Good afternoon ! Tell me the button to start measurements is not active; the program itself sees the device. Is this not a modification of the program?
> > 
> 
> > Hi! The button to start the measurements will become active after all the settings are configured correctly.
> > 
> 
> > —
> > Reply to this email directly, view it on GitHub, or unsubscribe.
> > You are receiving this because you are subscribed to this thread.

Yes, thanks! If the drivers for the scope are not correctly installed, you should see in the program terminal "HDS2202S not found, please try again..." when pressing the "Search and Setup DSO" button, and the button for starting the measurements will remain deactivated.

**Comment by mikhailmihalkov** [comment]
![123](https://github.com/SimoneAlbano000/OWON-HDS2202S-Bode-Plots/assets/99812383/b4643814-0bab-4d4e-8233-7670242cba06)
The original software works and the driver is installed correctly

**Comment by SimoneAlbano000** [comment]
I think i might have found the problem. Try the new release and let me know if the problem get's solved.

**Comment by mikhailmihalkov** [comment]
> I think i might have found the problem. Try the new release and let me know if the problem get's solved.

Like the previous version, the device finds it, but the measurement button is not active
